libvirt/CONTRIBUTING.md

```diff
@@ -83,6 +83,12 @@ We use the following tools:
        1    debian10   running
       ```
 
+### How to bump libvirt's version
```

Comment:
Yes, checkout https://pypi.org/project/virt-stats/
It's been under the radar because it is released with the same version as machine stats (because it's the same workflow). So you'll see a lot of virtstat versions without any changes (and yet a new version) because those versions were only intended for machine stats.